### DISCUSSION TITLE
Any interested in a documentation site (as opposed to just GitHub)?

### DISCUSSION THREAD

**Discussion by riceboyler** [TanStack/table#2007] [General]
https://react-table-docz.netlify.com/

I built this using Docz and I'm more than happy to share the codebase. I think provides a little better organization and navigation through the documentation, and it also allows for MDX, which could make examples REALLY powerful (see the Quick Start page I added for an example).

**Comment by tannerlinsley** [comment]
I would love to see a pull-request to add this to the repo. We can discuss more there!

**Comment by riceboyler** [comment]
Cool, I had a PR ready to go with it and then changed all kinds of stuff. :)

I'll get one ready tomorrow.

**Comment by riceboyler** [comment]
@tannerlinsley https://github.com/tannerlinsley/react-table/pull/2013
